docs/basics/functions.md

```diff
@@ -4,66 +4,63 @@ Flex functions can be combined in different ways to help you manipulate and tidy
+  - [Function precedence order](#function-precedence-order)
+  - [Flex supported functions](#flex-supported-functions)
+    - [add_attribute](#add_attribute)
+    - [convert_space](#convert_space)
+    - [ignore_output](#ignore_output)
+    - [keep_keys](#keep_keys)
+    - [lazy_flatten](#lazy_flatten)
+    - [lookup_file](#lookup_file)
+    - [math](#math)
+    - [perc_to_decimal](#perc_to_decimal)
+    - [remove_keys](#remove_keys)
+    - [rename_keys / replace_keys](#rename_keys--replace_keys)
+    - [sample_filter](#sample_filter)
+    - [sample_include_filter](#sample_include_filter)
+    - [sample_exclude_filter](#sample_exclude_filter)
+    - [snake_to_camel](#snake_to_camel)
+    - [split_array (leaf_array)](#split_array-leaf_array)
+    - [split_objects](#split_objects)
+    - [start_key](#start_key)
+    - [store_variables](#store_variables)
+    - [lookups](#lookups)
+    - [dedupe_lookups](#dedupe_lookups)
+    - [store_lookups](#store_lookups)
+    - [strip_keys](#strip_keys)
+    - [timestamp](#timestamp)
+    - [to_lower](#to_lower)
+    - [value_parser](#value_parser)
+    - [value_transformer](#value_transformer)
 
 ## Function precedence order
 
 Flex applies data parsing and transformation functions in a specific order, regardless of where in the configuration files you declare them. Keep the functions precedence order in mind to avoid unexpected or empty results.
 
--   [Data parsing and transformation functions](#data-parsing-and-transformation-functions)
-    -   [Function precedence order](#function-precedence-order)
-    -   [Flex supported functions](#flex-supported-functions)
-        -   [add_attribute](#addattribute)
-        -   [convert_space](#convertspace)
-        -   [ignore_output](#ignoreoutput)
-        -   [keep_keys](#keepkeys)
-        -   [lazy_flatten](#lazyflatten)
-        -   [lookup_file](#lookupfile)
-        -   [math](#math)
-        -   [perc_to_decimal](#perctodecimal)
-        -   [remove_keys](#removekeys)
-        -   [rename_keys / replace_keys](#renamekeys--replacekeys)
-        -   [sample_filter](#samplefilter)
-        -   [sample_include_filter](#sampleincludefilter)
-        -   [sample_exclude_filter](#sampleexcludefilter)
-        -   [snake_to_camel](#snaketocamel)
-        -   [split_array (leaf_array)](#splitarray-leafarray)
-        -   [split_objects](#splitobjects)
-        -   [start_key](#startkey)
-        -   [store_lookups](#storelookups)
-        -   [strip_keys](#stripkeys)
-        -   [timestamp](#timestamp)
-        -   [to_lower](#tolower)
-        -   [value_parser](#valueparser)
-        -   [value_transformer](#valuetransformer)
+  1. [lookup_file](#lookup_file)
+  2. [start_key](#start_key)
+  3. [strip_keys](#strip_keys) \*
+  4. [lazy_flatten](#lazy_flatten)
+  5. [split_array (leaf_array)](#split_array)
+  6. [split_objects](#split_objects)
+  7. Standard flatten (auto-flattening)
```

Comment:
Is missing the link?

Reply:
this is an internal function that is automatically applied so there is no documentation link here